build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -51,6 +56,7 @@
     "@llvm-project//mlir:ShapeTransforms": ["MLIRShapeOpsTransforms"],
     "@llvm-project//mlir:ToLLVMIRTranslation": ["MLIRTargetLLVMIRExport"],
     "@llvm-project//mlir:mlir-translate": ["mlir-translate"],
+    "@llvm-project//mlir:MlirLspServerLib": ["MLIRLspServerLib"],
```

Comment:
*sigh*. We should strip a leading "Mlir" from the Bazel targets in the conversion (and also fix this upstream)